tools/bin/mbedtls-archive-epic.py

```diff
@@ -0,0 +1,340 @@
+#!/usr/bin/env python3
+""" Assist with the transition of a projectv1 to a projectv2. 
+    This is currently used for management purposes and archiving EPICS.
+
+    It will only move public issues/pr's that have been closed/merged.
+
+    Requires a fully woking gh cli https://cli.github.com/ set-up
+    and an access token with the appropriate permissions.    
+"""
+# Copyright The Mbed TLS Contributors
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License") you may
+# not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import sys
+import json
+import requests
+from shlex import split
+from string import Template
+from subprocess import Popen, check_output, PIPE
+from pprint import pprint
+
+token = os.environ['GITHUB_TOKEN']
```

Comment:
Please support `gh auth token` if the environment variable is unset.

Also please arrange for `mbedtls-archive-epic.py --help` to work even when no token is available.